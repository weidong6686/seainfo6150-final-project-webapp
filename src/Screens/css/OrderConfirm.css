
.content{
    text-align: center;
    color:rgb(100, 94, 94)
}

.title{
    color: black;
    text-align: center;
}