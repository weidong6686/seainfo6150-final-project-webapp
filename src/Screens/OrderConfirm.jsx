import React from 'react'
import './css/OrderConfirm.css'
const OrderConfirm = (props) => {
    return (
        <div>
            <div className="content">
                <h2 className="title">Congratulations! Your order has been placed</h2>
                <p>Order Number: 14232876151</p>
                <p>Order Status: Confirmed</p>
            </div>
        </div>
    )
}

export default OrderConfirm