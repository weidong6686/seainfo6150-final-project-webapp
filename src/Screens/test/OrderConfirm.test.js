import React from "react";
import OrderConfirm from "../OrderConfirm.jsx";

describe("OrderConfirm tests", () => {
  it("renders correctly", () => {
    const { container } = (<OrderConfirm />);
    expect(container).toMatchSnapshot();
  });
});