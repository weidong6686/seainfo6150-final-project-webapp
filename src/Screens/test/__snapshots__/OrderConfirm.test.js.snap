// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderConfirm tests renders correctly 1`] = `undefined`;
